src/create-element.js

```diff
@@ -13,10 +13,7 @@ export function createElement(type, props, children) {
 	props = assign({}, props);
 
 	if (arguments.length>3) {
-		children = [children];
-		for (let i=3; i<arguments.length; i++) {
-			children.push(arguments[i]);
-		}
+		children = [].slice.call(arguments, 2);
```

Comment:
As @developit said, the best change we could have right now, would be these:

Start from 2, with and empty `children`
```suggestion
		children = [];
		for (let i=2; i<arguments.length; i++) {
			children.push(arguments[i]);
		}
```

Reply:
And also change the condition inside the if to `arguments.length>2`

Reply:
This shaves `-4B` 😃

Reply:
`arguments.length>2` => `arguments.length>2` will effect the next `if`.
```js
if (children!=null) {
    props.children = children;
}
```
Children never be null.

Reply:
right, I'd leave the other conditional as-is.